private/bufpkg/bufmodule/bufmoduleref/module_reference.go

```diff
@@ -15,6 +15,7 @@
 package bufmoduleref
 
 import (
+	"github.com/bufbuild/buf/private/bufpkg/bufconnect"
```

Comment:
bufmoduleref should not depend on bufconnect, bufmoduleref is a foundational package